### ISSUE TITLE
Clicking the Editor menu's doesn't work properly. 

### ISSUE THREAD

**Issue by EmptySet30** [godotengine/godot#36985]
**Godot version:**
3.2.0 and 3.2.1
**OS/device including version:**
Windows 10 x64
**Issue description:**
Clicking the editor's menu (eg. Scene and Import Buttons) doesn't respond as soon as you clicked it. the buttons will work only once you move your mouse away from the button. I don't have this problem in Godot 3.1.2. Btw, I'm using Laptop Dell Inspiron 14 
**Steps to reproduce:**
- Open the Editor
- Click a button (eg. Scene, Import, Node, Inspector, etc)
- The input doesn't respond

![screenshot](https://user-images.githubusercontent.com/61649928/76422862-dbd1b800-63e0-11ea-9445-aad9ecc5f288.jpg)

**Comment by pycbouh** [comment]
By "using laptop" do you mean that you use a touchpad, and not a mouse? If so, do you have this issue with a mouse?

**Comment by Calinou** [comment]
Duplicate of #23069. Try updating your graphics driver to the latest version *provided by Intel*. Alternatively, start the Godot editor with the `--verbose` command-line argument, which can also mitigate the issue.

**Comment by EmptySet30** [comment]
> Duplicate of #23069. Try updating your graphics driver to the latest version _provided by Intel_. Alternatively, start the Godot editor with the `--verbose` command-line argument, which can also mitigate the issue.



> Duplicate of #23069. Try updating your graphics driver to the latest version _provided by Intel_. Alternatively, start the Godot editor with the `--verbose` command-line argument, which can also mitigate the issue.

Thanks, My driver is updated and I checked issue #23069, apparently, Turning Show Update Spinner solves the problem. Thanks again!
